ui/src/assets/docs/de/developer-portal-page.md

```diff
@@ -0,0 +1,364 @@
+
+Um Kontodaten abzufragen, ersetzen Sie die IBAN in Ihrem Request mit Ihrer gewünschten IBAN von einem der definierten PSUs.
+
+`GET https://sandbox-api.dev.adorsys.de/v1/accounts`
+
+Der nachfolgende Code beschreibt einen beispielhaften cURL command, der alle Kontodaten des PSU "PSU-Successful" abfragt:
+
+```
+curl -v "https://sandbox-api.dev.adorsys.de/v1/accounts" \
+  -H "accept: application/json" \
+  -H "X-Request-ID: 99391c7e-ad88-49ec-a2ad-99ddcb1f7721" \
+  -H "Content-Type: application/json" \
+  -H "consent-id: I58hV2nWPJVJEvuw0dzl8qBkGcz40Qo_BCd_CjnTf_vsx7DeU-pL5sFaqwNUzbAThuXzrcFlVLs6eEVHdoFgKQ==_=_bS6p6XvTWI" \
+  -H "tpp-redirect-uri: https://adorsys.de/" \
+  `# get these two files from the PSD2 Accelerator certificate generator` \
+  --cert certificate.pem \
+  --key private.key \
+```
+
+Der nachfolgende Code beschreibt eine beispielhafte Response für eine erfolgreiche Abfrage der Kontodaten:
+
+```
+{
+  "accounts": [
+      {
+          "resourceId": "8660d175-2c79-4b68-a175-93b1866dc7e3",
+          "iban": "DE11760365688833114935",
+          "bban": "",
+          "msisdn": "",
+          "currency": "EUR",
+          "name": "",
+          "product": "Current Account",
+          "cashAccountType": "CACC",
+          "status": null,
+          "bic": "",
+          "linkedAccounts": "",
+          "usage": null,
+          "details": "",
+          "balances": [
+              {
+                  "balanceAmount":  {
+                      "currency": "EUR",
+                      "amount": "1500"
+                  },
+                  "balanceType": null,
+                  "lastChangeDateTime": null,
+                  "referenceDate": null,
+                  "lastCommittedTransaction": null
+             }
+          ],
+          "_links":  {
+              "viewTransactions": "https://sandbox-api.dev.adorsys.de/v1/accounts/8660d175-2c79-4b68-a175-93b1866dc7e3/transactions"
+       }
+    }
+  ]
+}
+```
+
+| PSU-ID                  | Iban                     | Consent Status |
+| :---------------------- | :----------------------- | :------------- |
+| PSU-Successful          | DE11760365688833114935\* | valid          |
+| PSU-ConsentExpired      | DE12760365687895439876   | expired        |
+| PSU-ConsentRevokedByPsu | DE89760365681729983660   | revokedByPsu   |
+
+(\*) PSU-Successful beinhaltet weitere Konten, um das folgende Verhalten testen zu können: 
+
+- Ein Konto verfügt über einen negativen Buchungssaldo und einen abweichenden verfügbaren Kontostand (Ein verfügbarer Kontostand berücksichtigt vorgemerkte Umsätze, die noch nicht gebucht wurden, wie beispielsweise Kartenzahlungen)
+  (DE89760365681134661389)
+- Ein Konto beinhaltet keine Transaktionen und der verfügbare Kontostand ist 0 
+  (DE07760365680034562391)
+- Ein Konto hat einen geringeren verfügbaren Kontostand als der aktuelle Buchungssaldo. 
+  (DE71760365681257681381)
+- Die Währung eines Kontos lautet auf USD (DE56760365681650680255)
```

Comment:
(DE89760365681134661389)
- Ein Konto enthält keine Transactions und die Balance ist 0 
  (DE07760365680034562391)
- Ein Konto hat eine geringere verfügbare Balance als der aktuelle Buchungssaldo. 
  (DE71760365681257681381)
- Die Währung eines Kontos lautet ist in USD angegeben (DE56760365681650680255)

Reply:
done